src/libraries/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -1160,6 +1163,88 @@ public void ThreadPoolMinMaxThreadsEventTest()
             }).Dispose();
         }
 
+        private sealed class RuntimeEventListener : EventListener
+        {
+            private const string ClrProviderName = "Microsoft-Windows-DotNETRuntime";
+            private const EventKeywords ThreadingKeyword = (EventKeywords)0x10000;
+
+            public volatile int TPIOEnqueue = 0;
+            public volatile int TPIODequeue = 0;
+            public ManualResetEvent TPWaitIOEnqueueEvent = new ManualResetEvent(false);
+            public ManualResetEvent TPWaitIODequeueEvent = new ManualResetEvent(false);
```

Comment:
Based on typical naming convention:

```suggestion
            public volatile int tpIOEnqueue = 0;
            public volatile int tpIODequeue = 0;
            public ManualResetEvent tpWaitIOEnqueueEvent = new ManualResetEvent(false);
            public ManualResetEvent tpWaitIODequeueEvent = new ManualResetEvent(false);
```